src/cephadm/cephadm

```diff
@@ -9461,9 +9461,9 @@ def _get_parser():
         action='store_true',
         help='Do not automatically provision monitoring stack (prometheus, grafana, alertmanager, node-exporter)')
     parser_bootstrap.add_argument(
-        '--skip-centralized-logging',
+        '--add-centralized-logging',
```

Comment:
this sounds better to me but if everyone's okay with `add` then I am good

```suggestion
        '--with-centralized-logging',
```

Reply:
Done.